content/how-to/rackspace-data-protection/data-protection-patching/index.md

```diff
@@ -0,0 +1,16 @@
+---
+permalink: "data-protection-patching"
+audit_date: '2022-10-26'
+type: article
+title: "Rackspace Data Protection Patching"
+created_date: '2022-10-21'
+created_by: Asmita Nakwa
+last_modified_date: '2022-10-26'
+last_modified_by: Asmita Nakwa
+product: Rackspace Data Protection
+product_url: rackspace-data-protection
+---
+
+# Rackspace Data Protection Patching
+
+In order to ensure the best quality of service, periodic patching of the Rackspace Data Protection infrastructure is required. However, patching does not require downtime and should be non-disruptive to normal backup and restore opportunities. During updates, nodes are taken down one at a time, with all operations failing over to other active nodes. Due to the higher potential of an unanticipated impact, Rackspace will inform customers of maintenance windows; changes, however, are projected to have no impact.
```

Comment:
This seems different from the word dow. What is the benefit of changing to this?

Reply:
Retain as word doc.